src/bindings/python/tests/test_runtime/test_model.py

```diff
@@ -560,6 +560,23 @@ def check_rt_info(model, serialized):
         assert model.get_rt_info(["config", "model_parameters", "labels", "label_groups", "ids"]) == ids
         assert model.get_rt_info(["config", "model_parameters", "mean_values"]) == mean
 
+        rt_info = model.get_rt_info()
+        assert rt_info["config"] == {"converter_type": "classification",
+                                     "model_parameters": {"labels": {"label_groups": {"ids": ids},
+                                                          "label_tree": {"directed": directed, "float_empty": empty,
+                                                                         "nodes": empty, "type": "tree"}},
+                                                          "max": max_val, "mean_values": mean,
+                                                          "min": min_val, "threshold": threshold},
+                                     "type_of_model": "classification"}
+
+        for key, value in rt_info.items():
+            if key == "config":
+                for config_value in value:
+                    assert config_value in ["type_of_model", "converter_type", "model_parameters"]
+
+        for rt_info_val in model.get_rt_info(["config", "model_parameters", "labels", "label_tree"]):
+            assert rt_info_val in ["float_empty", "nodes", "type", "directed"]
```

Comment:
I think you need to remove this if:
https://github.com/openvinotoolkit/openvino/pull/15636/files#diff-fc5bff420a93e90ef3bbea6d6b2bf55e76901d076d9584be0bcd98135479b4f3R523
In C++ we don't have difference between original and serialized models.

Reply:
done